connrequester/connrequester_nomodemd.go

```diff
@@ -0,0 +1,38 @@
+// +build linux/amd64
```

Comment:
This isn't correct (and is why the build is failing in Travis). Platforms and architectures can't be combined with a slash.

Perhaps just go with `+build amd64` and `+build !amd64` for the other file.

Reply:
The `_nomodemd.go` suffix is great. Much clearer.